mrp_operations_rejected_quantity/wizard/wiz_stop_production_operation.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# (c) 2016 Alfredo de la Fuente - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp import models, fields, api
+
+
+class WizStopProductionOperation(models.TransientModel):
+    _name = 'wiz.stop.production.operation'
+
+    employee_id = fields.Many2one(
+        comodel_name='hr.employee', string='Employee', required=True)
+    accepted_amount = fields.Integer(
+        string='Accepted amount', default=0)
+    rejected_amount = fields.Integer(
+        string='Rejected amount', default=0)
+    stop_date = fields.Datetime(
+        string='Date stop', default=fields.Datetime.now, required=True)
+
+    @api.model
+    def default_get(self, var_fields):
+        operation_obj = self.env['mrp.production.workcenter.line']
+        res = super(WizStopProductionOperation, self).default_get(var_fields)
+        operation = operation_obj.browse(self.env.context['active_id'])
```

Comment:
No habría que hacer una comprobación anterior "if 'active_id' in self.env.context:" antes de esta línea?